hloc/extract_features.py

```diff
@@ -8,12 +8,10 @@
 import numpy as np
 from tqdm import tqdm
 import pprint
-
```

Comment:
Great, thanks! Could you please remove all the unrelated line addition & deletion that show up in the diff? Thanks :)